lib/common.sh

```diff
@@ -67,7 +67,7 @@ export SUSHY_TOOLS_IMAGE=${SUSHY_TOOLS_IMAGE:-"quay.io/metal3-io/sushy-tools"}
 
 # Ironic vars
 export IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
-export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
+export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/stbenjam/ironic-image:latest"} # FIXME: Built from 'ipv6' branch on my fork
```

Comment:
Tangential to the PR but I wonder if we could refactor and reuse the local image stuff @derekhiggins added to dev-scripts to make testing with custom image branches easier